Description: Disable gconf_engine_key_is_writable
 This symbol is not available in libgconf when it is configured without
 ORBit support, which is currently the case in Debian.
Author: Colin Watson <cjwatson@ubuntu.com>
Bug-Debian: http://bugs.debian.org/649055
Forwarded: no
Last-Update: 2011-11-17

Index: b/xs/GConfEngine.xs
===================================================================
--- a/xs/GConfEngine.xs
+++ b/xs/GConfEngine.xs
@@ -350,6 +350,8 @@
	if (err)
		gperl_croak_gerror (NULL, err);

+#if 0 /* requires gconf to be configured with ORBit support */
+
 ##gboolean gconf_engine_key_is_writable  (GConfEngine *conf,
 ##                                        const gchar *key,
 ##                                        GError     **err);
@@ -366,6 +368,8 @@
     OUTPUT:
         RETVAL

+#endif
+
 guint
 gconf_engine_notify_add (engine, namespace_section, func, data=NULL)
	GConfEngine * engine
